src/main/java/core/mate/academy/service/ExcavatorProducerImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ExcavatorProducerImpl implements MachineProducer<Excavator> {
+    private List<Excavator> excavators;
+
+    public ExcavatorProducerImpl() {
+        excavators = new ArrayList<>();
+    }
```

Comment:
Don't begin method implementation with an empty line. Remove the empty line inside the constructor.